File: apps/console/BasketballConsole/BasketballConsole/Models/CosmosDB/ValueObjects/GamePlay.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace NBA.Models.CosmosDB
{
    public class GamePlay
    {
        public int PlayerId { get; set; }
        public int Quarter { get; set; }
        public string Type { get; set; }
        public int? Points { get; set; }
        public TimeSpan At { get; set; }
    }
}

File: WebSite with CosmosDB/client/nba-statistics/src/models/GamePlay.ts
export class GamePlay {
    quarter: number;
    type: string;
    points?: number | null;
    at: Date;

    constructor(quarter: number, type: string, points: number | null | undefined, at: Date) {
        this.quarter = quarter;
        this.type = type;
        this.points = points ?? null;
        this.at = at;
    }
}
